### ISSUE TITLE
citrix_adc_service fails in clustered mode on default-path-monitor

### ISSUE THREAD

**Issue by vkovacik** [citrix/citrix-adc-ansible-modules#124]
**Issue:**
`citrix_adc_service` module fails on service which has path monitoring enabled. 

Error:
```
fatal: [REDACTED -> localhost]: FAILED! => changed=false 
  loglines:
  - Applying actions for state present
  - Deleting binding for monitor default-path-monitor
  msg: nitro exception errorcode=257, message=Operation not permitted
```

**Investigation:**
Path monitoring binds `default-path-monitor` to the service. 

```
show service REDACTED
...
	Path Monitoring: YES
	Individual Path Monitoring: YES
...
1)	Monitor Name: tcp-default
		State: UP	Weight: 1	Passive: 0
		Probes: 3482171	Failed [Total: 1204 Current: 0]
		Last response: Success - TCP syn+ack received.
		Response Time: 0.0 millisec
2)	Monitor Name: default-path-monitor
		State: UP	Weight: 65535	Passive: 0
		Probes: 3778683	Failed [Total: 95 Current: 0]
		Last response: Success - ICMP echo reply received.
		Response Time: 2.650 millisec
```

**Goal:**
Add `default-path-monitor` to the list of internal monitors to ignore:
https://github.com/citrix/citrix-adc-ansible-modules/blob/fed898d7a73fadcc4023237b8b7dc85f90327789/ansible-modules/citrix_adc_service.py#L456

Documentation on path monitoring in clustered mode:
https://docs.citrix.com/en-us/citrix-adc/13/clustering/cluster-usage-scenarios/path-monitoring-in-cluster.html#to-enable-path-monitoring-for-servicesservice-groups-by-using-the-gui

**Comment by vkovacik** [comment]
PR: https://github.com/citrix/citrix-adc-ansible-modules/pull/125

**Comment by giorgos-nikolopoulos** [comment]
Fixed with #178

There is now a new attribute `ignore_monitors` which accepts a list of monitors to ignore when reading the service and comparing local and remote states.

The default value contains the `default-path-monitor`.
